.github/workflows/test-container-image-tags.yml

```diff
@@ -47,6 +48,28 @@ jobs:
           source 'assert.sh'
           assert_eq ${{ env.VERSION }} "main" "not equivalent!"
 
+  build-container-image-tags-action-dist:
+    runs-on: ubuntu-latest
+    name: Test container image tags for branch
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - name: Build with ncc
+        run: |
+          cd container-image-tags
+          npm install -g yarn
```

Comment:
Not sure if this necessary. AFAIK yarn is already installed on the ubuntu machines.

Reply:
can confirm.
Also you can find a workaround for caching the node modules here: https://github.com/greenbone/gsa/blob/main/.github/workflows/ci-js.yml#L21

Reply:
this is not on all runner the case like sometimes on our own self-hosted